smart-client-core/src/main/java/com/emc/rest/smart/SmartConfig.java

```diff
@@ -132,6 +138,18 @@ public Object getProperty(String propName) {
         return properties.get(propName);
     }
 
+    public Integer getIntProperty(String propName, int defaultValue) {
+        int ret;
+        String pValue = (String)properties.get(propName);
+        try {
+            ret = Integer.parseInt(pValue);
+        } catch (RuntimeException e) {
+            log.debug("cannot parse the config value for {}", propName);
```

Comment:
sorry, but we need the exception here too - you'll have to concatenate the `propName` and pass the exception as the only parameter (to get the log to print the stack trace)